src/main/java/de/dataelementhub/rest/controller/v1/ExportController.java

```diff
@@ -0,0 +1,96 @@
+package de.dataelementhub.rest.controller.v1;
+
+import de.dataelementhub.model.dto.export.ExportInfo;
+import de.dataelementhub.model.dto.export.ExportRequest;
+import de.dataelementhub.model.service.ExportService;
+import de.dataelementhub.rest.DataElementHubRestApplication;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.core.annotation.Order;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1")
+public class ExportController {
+
+  private final ExportService exportService;
+
+  @Autowired
+  public ExportController(ExportService exportService) {
+    this.exportService = exportService;
+  }
+
+  public static String exportDirectory = System.getProperty("user.dir") + "/uploads/export/";
```

Comment:
Please change from user.dir to a configurable directory that can be chosen via docker-compose/.env files.